apps/lcs/translations/src/en/validation.json

```diff
@@ -42,7 +42,8 @@
   },
   "extra-tenant-tel": {
     "required": "Enter a telephone number",
-    "telephoneNumber": "Telephone number must be between 8 and 16 characters"
+    "telephoneNumber": "Telephone number must be between 8 and 16 characters",
```

Comment:
In this case, I would suggest using another validators such as max length, min length.

Reply:
After speaking with UCD this is an international phone number, max and min length are not required as the logic is built into 'internationalPhoneNumber'

Reply:
internationalPhoneNumber - add the error message for this particular type.